src/main/webapp/common.js

```diff
@@ -15,6 +15,7 @@
 /**
  * Common methods for both the app page.
  */
+var APIKey = "";
```

Comment:
Consider a future where this code will call many APIs.

Maybe call this GoogleMapsApiKey?

Reply:
I renamed it to GoogleMapsApiKey.